base-template/about-base-template.md

```diff
@@ -11,84 +24,349 @@ The base template goes one step further, in that it helps you create any templat
+* If so, reference your style guide which should set recommendations on image and screenshot size, video length, style etc.
+
+}
+
+## Priorities for each quality criteria
+
+{Template author tip: 
+
+* Each doctype addresses different use cases. It target’s different user personas, with different needs.
+* As such, there are different weightings applied to quality characteristics for each doctype. Reference documentation needs to comprehensively address all the features in the latest version, but can tolerate lower quality writing. Tutorials need only cover a sample of use cases of an older version of the software, but there is a greater need for writing quality.}
+
+{Docset owner tip: Select the quality criteria which will be important for your project.}
+
+The following table lists the relative importance of the following quality criteria for {doctype} documents.
+
+
+|Quality criteria|Priority to address|
+|--- |--- |
+|Currency: Alignment with the latest version of software.|{Select one of: Must, Should, Nice to have (May), Not Applicable (N/A), Should not}|
+|Currency: Alignment with a specified version of software.||
+|Comprehensive: Covers every feature and use case.||
+|Well written: Unambiguous.||
+|Well written: Concise, high signal to noise ratio, high information density.||
+|Well written: Aligns with writing style guide.||
+|Well written: Engaging, entertaining, draws the reader in.||
+|Audience: Target a specific persona and user story. Simplifies documentation by assuming prerequisite knowledge of the reader.||
+|Audience: Targets a broad international, multicultural audience. Avoids slang, colloquialisms, pop cultural references or local references.||
+|Audience: Considers a non-English speaking audience. Uses simple English.||
+|Audience: Consider accessibility and disability in audiences.||
+|Audience: Consider diversity and inclusiveness in language chosen.||
+|Maintainability: Written to be easily translated.||
+|Maintainability: Uses timeless language with reduced need to update over time.||
+
+Note: The terms “Must”, “Should”, and “May” should be interpreted as per [RFC 2119.](https://www.ietf.org/rfc/rfc2119.txt) 
+
+_TBD: The quality criteria list requires review and refinement._
+
+## Implementation strategy
+
+{Template author tip:
+
+* This section is optional.
+* In some cases there may be tips you can provide on how to create a specific doctype.
+
+}
+
+## Customization
+
+{Template author tip:
+
+* There are often multiple approaches to consider when implementing a doctype.
+* In this section you should discuss different approaches that can be applied to this doctype, including logic and recommendations to help a docset owner select the best strategy for them.
+* Often there would be a light version of this template to use for immature projects, and a more comprehensive version for mature projects. You should discuss how a project might mature their docs from light to comprehensive versions.
```

Comment:
Valid point. The point I was thinking is more about the capacity of the project to produce documentation, rather than the need for the docs. I've tweaked text to:

* Often there would be a light and more comprehensive version of this template. Immature projects may only have the capacity to maintain the light version. You should discuss how a project might mature their docs from light to comprehensive versions.